LSP-css.sublime-settings

```diff
@@ -7,21 +7,112 @@
 				"scopes": ["source.css"],
 				"syntaxes": [
 					"Packages/CSS/CSS.sublime-syntax",
-					"Packages/CSS3/CSS3.sublime-syntax"
-				]
+					"Packages/CSS3/CSS3.sublime-syntax",
+				],
 			},
 			{
 				"languageId": "scss",
 				"scopes": ["source.scss"],
-				"syntaxes": ["Packages/Sass/Syntaxes/SCSS.sublime-syntax"]
+				"syntaxes": ["Packages/Sass/Syntaxes/SCSS.sublime-syntax"],
 			},
 			{
 				"languageId": "less",
 				"scopes": ["source.less"],
-				"syntaxes": ["Packages/LESS/Syntaxes/LESS.sublime-syntax"]
-			}
+				"syntaxes": ["Packages/LESS/Syntaxes/LESS.sublime-syntax"],
+			},
 		],
-		"initializationOptions": {},
-		"settings": {}
-	}
-}
\ No newline at end of file
+		"initializationOptions": {
+			"dataPaths": [],
```

Comment:
I can't see any references to `dataPaths` in https://github.com/Microsoft/vscode-css-languageservice

Reply:
I encountered the following error if it's not specified.

```
error: Request initialize failed with message: Cannot read property 'dataPaths' of undefined
lsp-css for folderless window did not initialize properly. Disabling until this window is restarted
RuntimeError: Request initialize failed with message: Cannot read property 'dataPaths' of undefined
```

it's supposed to happen in https://github.com/jfcherng/LSP-css/blob/7225f60d2434fa219db76631364794875cb7ef71/vscode-css/out/cssServerMain.js#L87. so more accurately speaking, it seems to be because we are not sending `initializationOptions` (i.e., it's `null` in the executed js server) if it's an empty dict.

Reply:
Right, I just came to the same conclusion before seeing your comment.
I think it's a bug in Sublime LSP but then we should keep that setting for now.

Reply:
issue filed https://github.com/sublimelsp/LSP/issues/917